bundler/lib/bundler/inline.rb

```diff
@@ -51,6 +51,8 @@ def gemfile(install = false, options = {}, &gemfile)
       definition = builder.to_definition(nil, true)
       def definition.lock(*); end
       definition.validate_runtime!
+      
+      rubygems_deprecate(install)
```

Comment:
Thanks for the reply @deivid-rodriguez. I'll work on it.

Reply:
I'm trying to run the `inline_spec.rb` individually with the following command `bundler/bin/rspec ./bundler/spec/runtime/inline_spec.rb`, but I'm getting `undefined method `build_lib'` there are something that I'm missing?

Reply:
Cd to the `bundler` folder first, and run `bin/rspec` from there.

Reply:
Thanks!

Reply:
Yes, if you pass "3", then nothing changes because the helper will only applies when Bundler version is 3, which is not yet.

Can you paste the failures?

Reply:
And upload the code you've written so far?

Reply:
I added this line, like you suggested.
```ruby
 Bundler::SharedHelpers.major_deprecation 2, "The 'install' argument is deprecated." if install
```
And this are the failures:
`rspec ./spec/runtime/inline_spec.rb:48 # bundler/inline#gemfile requires the gems`
`rspec ./spec/runtime/inline_spec.rb:242 # bundler/inline#gemfile does not leak Gemfile.lock versions to the installation output`
`rspec ./spec/runtime/inline_spec.rb:423 # bundler/inline#gemfile when requiring fileutils after does not show redefinition warnings`
`rspec ./spec/runtime/inline_spec.rb:345 # bundler/inline#gemfile skips platform warnings`

The logs are very long, so I hope this would be enough.

Reply:
That just says which specs are failing, but not the errors they are throwing. You can copy just one, or upload the logs to a gist, or even better, push the code so that we can see the failures in CI.

Reply:
Ok. I'll push the code.

Reply:
Should I rewrite those tests? Also, I'm writing one that shows the deprecated message.

```ruby
  it "shows platform warnings" do
    script <<-RUBY
      gemfile(true) do
        source "#{file_uri_for(gem_repo1)}"
        gem "rack", platform: :jruby
      end
    RUBY

    expect(err).to eq("[DEPRECATED] The 'install' argument is deprecated.")
  end
  ```

Reply:
There's a helper called `err_without_deprecations` that you can use instead of `err` to get those specs passing. And then you can add an extra spec for the deprecation message.